<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(207, 207, 207);
background-color: rgb(232, 232, 232);</string>
  </property>
  <widget class="QLineEdit" name="lineEditCargarArchivo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCargarArchivo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Buscar y Cargar archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCvGn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Generar Paleta de Coleres</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelArchivos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>92</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Archivos .html</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TABLA DE FRECUENCIA DE COLORES</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_Colores">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>121</width>
     <height>371</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Color</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tbl_Colores_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>121</width>
     <height>371</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Frecuencia</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
